scripts/linux-prereqs.sh

```diff
@@ -85,7 +85,12 @@ elif type apt > /dev/null 2>&1; then
             echo ""
             echo "(!) WARNING: $LIBSSLCOUNT sub-versions of libssl1.0 detected. This can crash Live Share."
             echo ""
-            read -p "Attempt to fix by removing $LIBSSLFIRSTPKG [Y/n]? "
+            echo "This script can attempt to fix this by removing the package \"$LIBSSLFIRSTPKG\"".
+            echo "However, doing so MAY REMOVE OTHER PACKAGES ON YOUR SYSTEM. If you proceed, you"
```

Comment:
Instead of being presented with 2 user actions, can we make this as The script can attempt to fix this by remove package \"$LIBSSLFIRSTPKG\", however doing that may also cause the following packages to be removed, and some list to be added

Reply:
I'll need to see if there's a good way to get that information reliably.  The apt command handles all of that and presents a confirmation itself which is what this refers to.  At this point in the flow we don't actually know the complete list of packages the OS's package manager will remove if libssl1.0.x is removed.

Reply:
I think I'll look at this as a second PR as I think about it since I want to be sure this fix is in.  This change is pretty isolated.  I'd like to be sure this update makes it into our next release.